### ISSUE TITLE
IE unit tests are broken on CI

### ISSUE THREAD

**Issue by jason0x43** [theintern/intern#1037]
This is an issue with the current development version of Intern (5.0-pre).

From looking at BrowserStack logs, the problem seems to be that the remote test runner can't communicate with the host Intern. CI tests run fine on Chrome, though, so it's not a problem with the basic test setup.

Note that functional tests do work, so the host Intern can communicate with the remote IE and and send it commands.

**Comment by jason0x43** [comment]
The problem may be that remote execution scripts are being transpiled to ES2015 code.

```
× internet explorer 11 on WINDOWS - functional/webdriver/helpers/pollUntil - with args (2.769s)
    JavaScriptError: [POST https://(redacted)@hub.browserstack.com:443/wd/hub/session/1e5f14caa4488a5409ed83a6c10387215201835b/execute_async / {"script":"return (function(poller,args,timeout,pollInterval,done){/* jshint evil:true */poller=new Function(poller);const endTime=Number(new Date())+timeout;(function poll(){const result=poller.apply(this,args);/*jshint evil:true */if(result!=null){done(result);}else if(Number(new Date())<endTime){setTimeout(poll,pollInterval);}else{done(null);}})();}).apply(this, arguments);","args":["return (id => {\n                    return document.getElementById(id);\n                }).apply(this, arguments);",["d"],1000,67]}] JavaScript error in async script.
```

Note the arrow function in that code.

**Comment by jason0x43** [comment]
Resolved by a681b7f
